vendor-bin/phpstan/composer.json

```diff
@@ -0,0 +1,9 @@
+{
+  "require": {
+    "php": "^7.3 || ^8.0",
+    "phpstan/phpstan": "^0.12.100"
```

Comment:
Some of these versions can be bumped. The `mot-logger` PR should have the latest